jekyll/_cci2/circleci-images.md

```diff
@@ -322,7 +299,7 @@ or [Stretch](https://packages.debian.org/stretch/)-based images,
 however the next-gen images, `cimg`, extend the official [Ubuntu](https://packages.ubuntu.com) image.
 For details on the next-gen images, see the [Developer Hub](https://circleci.com/developer/images/). Each image is tracked in its own repository.
 
-The following packages are installed via `curl` or other means.
+The following packages are pre-installed in convenience images using `curl` or other means.
```

Comment:
This is the actual fix for the ambiguous language. The reader thought _they_ needed to install with `curl`.